### ISSUE TITLE
[French] Help output of cf commands should not be translated

### ISSUE THREAD

**Issue by fbiville** [cloudfoundry/cli#1169]
In short, it looks like there are some bad translations hanging out in the cf cli. Not too serious, but pretty confusing if it's your first time using it.

Example output of running `cf apps` in the fr_FR locale:
```
cf apps
ECHEC
Aucun noeud final d'API défini. Utilisez 'Connexion cf' ou 'API cf' pour cibler un noeud final.
```

## What occurred

It translated "cf login" and "cf api" commands, even flipping the order.

## What you expected to occur
Don't do that. Please.

## CLI Version  

cf version 6.27.0+d26b32dcc.2017-06-08

Enjoy!

**Comment by dkoper** [comment]
Thanks, we think were understand where the problem lies.

**Comment by dkoper** [comment]
This was fixed in today's 6.28.0 cf CLI release. Thanks again for reporting the issue.
https://github.com/cloudfoundry/cli#downloads

Regards,
Dies Koper
CF CLI PM

**Comment by tjarratt** [comment]
Aww dang, thanks for fixing that @dkoper. I actually wrote this issue using Florent's Github account while we were pairing a few weeks ago. It felt very strange to me, because I had originally worked on the i18n features in the CLI, and had contributed some of the very, very early translations.

Thank you (and the rest of the team) so much for fixing this so quickly. It really brought a big smile to my face to see the CLI again from this point of view, and to see the team iterating so quickly on features that provide value to end users.

Merci beaucoup!
